BLSQ/iaso
# POLIO-1034: move percentage of target population from risk assessment tab to rounds tab

Percentage covered target population should be in rounds tab instead of risk assessment tab

Related JIRA tickets : POLIO-1034

## Self proofreading checklist


## Changes

- remove field from Risk Assessment Tab and add it for each round sub tab 
- update translations

## How to test

- Go to campaigns
- Create or edit a campaign
- Check the field is in each round sub tab

## Print screen / video

https://github.com/BLSQ/iaso/assets/25134301/545ff250-7835-4002-a357-ca1406dff1c3
